test/fixtures/06-with-toolchain-override/vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "functions": {
+    "api/**/*.rs": {
+      "runtime": "vercel-dev-builder@0.0.5",
+      "includeFiles": "static/**/*.txt"
```

Comment:
If we have `includeFiles`, can't we 🗑️ the `build.sh` file?

Reply:
The `build.sh` file is writing the rust compiler version after the toolchain has been installed to the text file. I know it's a bit confusing as the file already has the contents `rustc 1.70.0-nightly (44f518058 2023-03-20)`